packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -272,6 +273,20 @@ const ruleset: any = {
     /// ARM RPC rules for Patch patterns
     ///
 
+    // RPC Code: RPC-Patch-V1-01
+    PatchPropertiesCorrespondToPutProperties: {
+      description:
+        "Patch request body MUST contain at least one or more properties present in the normal resource definition (PUT operation).",
```

Comment:
>Patch request body MUST contain at least one or more properties present in the normal resource definition (PUT operation). [](http://example.com/codeflow?start=9&length=122)

Patch request body MUST contain at least one or more properties present in the resource definition as also indicated in the request or response payload of a PUT operation.

Reply:
Assuming thats same as
"Patch request body must only contain properties present in the corresponding put request body, and must contain at least one property."